homeassistant/components/sensor/darksky.py

```diff
@@ -166,6 +182,9 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
     forecast = config.get(CONF_FORECAST)
     sensors = []
     for variable in config[CONF_MONITORED_CONDITIONS]:
+        if variable in DEPRECATED_SENSOR_TYPES:
+            _LOGGER.warning("Monitored condition %s is deprecated.",
```

Comment:
Just a suggestion, but you could change `DEPRECATED_SENSOR_TYPES` to be dictionary instead of a set and provide the "new" name for the condition (maybe even with a message that their new "interval" like `currently` or `hourly` won't be supported anymore).

Reply:
currently and hourly don't seem to ever have been supported. I'm not sure why they were in the code. A dictionary isn't a bad idea though.